### PR TITLE
Setup cleanup

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

File: forms/setupdialog.ui [hunk 1]
```diff
@@ -224,6 +239,9 @@ min-width: 200px;
    </item>
    <item>
     <widget class="QStackedWidget" name="wizardStackedWidget">
+     <property name="toolTip">
+      <string>Jump to machine key page</string>
+     </property>
```
Comment (shinnok) [inline forms/setupdialog.ui]: What is this for?

Comment (gperciva) [inline forms/setupdialog.ui]: Adding a voice description of the radio buttons?  I'll put it in a separate PR.

**Comment by gperciva** [comment]
Updated to make this purely about "cleanup" and not accessibility (that will go in a separate PR).

File: forms/setupdialog.ui (hunk 1 above)
Comment (shinnok) [inline forms/setupdialog.ui]: Why is it set on the wizardStackedWidget, the corresponding radioButton already has a tooltip set?

Comment (gperciva) [inline forms/setupdialog.ui]: Sorry, I must have clicked on the wrong GUI element when adding this text.  I'll fix this and double-check it before submitting the new PR.

**Pull request merged**